### PR TITLE
Lightweight testing of operator cli

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Comment by howardjohn** [comment]
This is failing only because the operator is actually broken for auto mtls

**Comment by howardjohn** [comment]
fyi @sdake @ostromart

File: tests/integration/tests.mk [hunk 1]
```diff
@@ -63,9 +63,13 @@ NEW_INSTALLER_TARGETS = $(shell GOPATH=${GOPATH} go list ../istio/tests/integrat
...
-test.integration.new.installer: | $(JUNIT_REPORT)
-	KUBECONFIG=${INTEGRATION_TEST_KUBECONFIG} kubectl apply -k github.com/istio/installer/kustomize/cluster
-	KUBECONFIG=${INTEGRATION_TEST_KUBECONFIG} kubectl apply -k github.com/istio/installer/test/demo
+test.integration.new.installer: istioctl | $(JUNIT_REPORT)
+	KUBECONFIG=${INTEGRATION_TEST_KUBECONFIG} ${ISTIO_OUT}/istioctl manifest apply \
+		--set hub=${HUB} \
+		--set tag=${TAG} \
+		--skip-confirmation \
+		--logtostderr \
+		--set values.global.imagePullPolicy=${_INTEGRATION_TEST_PULL_POLICY}
```
Comment (elfinhe) [inline tests/integration/tests.mk]: np: move all sets together?

Comment (elfinhe) [reply tests/integration/tests.mk]:
```
		--skip-confirmation \
		--logtostderr \
		--set hub=${HUB} \
		--set tag=${TAG} \
		--set values.global.imagePullPolicy=${_INTEGRATION_TEST_PULL_POLICY}
```

**Pull request merged**